server/form-pages/assess/reviewApplication/sufficientInformation/sufficientInformationSent.ts

```diff
@@ -16,18 +17,29 @@ export default class SufficientInformationSent implements TasklistPage {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   constructor(public readonly body: Record<string, unknown>) {}
 
-  user: ApprovedPremisesUser
+  userName: string
 
-  static async initialize(
-    body: Record<string, unknown>,
-    assessment: Assessment,
-    token: string,
-    dataServices: DataServices,
-  ) {
-    const user = await dataServices.userService.getUserById(token, assessment.application.createdByUserId)
+  emailAddress: string
+
+  phoneNumber: string
+
+  static async initialize(body: Record<string, unknown>, assessment: Assessment) {
+    const userName = retrieveQuestionResponseFromFormArtifact(assessment.application, ConfirmYourDetails, 'name')
+    const emailAddress = retrieveQuestionResponseFromFormArtifact(
+      assessment.application,
+      ConfirmYourDetails,
+      'emailAddress',
+    )
+    const phoneNumber = retrieveQuestionResponseFromFormArtifact(
```

Comment:
these values may not always exist so it would be safer to use `retrieveOptionalQuestionResponseFromFormArtifact` and provide a fallback so we don't have 'undefined' show up in the UI